CMakeLists.txt

```diff
@@ -3,6 +3,8 @@
 # CMake 3.21 is needed for file(IMPORTED_RUNTIME_ARTIFACTS)
```

Comment:
Unless I'm overlooking something, this removes IMPORTED_RUNTIME_ARTIFACTS, so we might be able to reduce the minimum cmake version again?

Reply:
Yep, I was leaving that for another commit.